kotlin/src/main/java/app/rive/runtime/kotlin/controllers/RiveController.kt

```diff
@@ -4,11 +4,11 @@ abstract class RiveController<T> {
     var isActive = false
         protected set
 
-    open fun initialize(core: T): Boolean {
+    open fun initialize(artboard: T): Boolean {
```

Comment:
Oh, I forgot to remove these Controllers from here, going to give it a pass next